Packs/Base/Scripts/StixParser/StixParser.py

```diff
@@ -34,6 +34,7 @@
 
 # Pattern Regexes - used to extract indicator type and value
 INDICATOR_OPERATOR_VAL_FORMAT_PATTERN = r"(\w.*?{value}{operator})'(.*?)'"
+INDICATOR_IN_VAL_PATTERN = r"(\w.*?valueIN)\(+('.*?')\)"
```

Comment:
this regex did not match the example in the test for me, got an idea why? shouldn't it be something like:
```suggestion
INDICATOR_IN_VAL_PATTERN = r"(\w*?value IN )\(+('.*?')\)"
```
not sure what is the exact pattern, please consult with Dror.

Reply:
line 243 - `trimmed_pattern = pattern.replace(" ", "")`

it really confused me when I looked at it 😅

Reply:
After consulting with Dror, the value I entered is correct (the spaces are removed before using the regex).
An additional comment was added